packages/help-center/src/types.ts

```diff
@@ -1,16 +1,13 @@
-import { SiteDetails } from '@automattic/data-stores/dist/types/site';
-import { ReactElement, ReactNode } from 'react';
+import { SiteDetails } from '@automattic/data-stores';
+import { ReactElement } from 'react';
```

Comment:
We are usually careful to include `type` keyword when importing types, like `import type { ReactElement }` or `import { type ReactElement }`. If it's missing, I've seen Babel getting confused about missing imports.